_rules/text-contrast-afw4f7.md

```diff
@@ -46,6 +46,8 @@ For each test target, the [highest possible contrast][] between the [foreground
 
 - Text that has the same foreground and background color (a contrast ratio of 1:1) is not considered to be "visual presentation of text", making it inapplicable to the success criterion. Text hidden in this way can still cause accessibility issues under other success criteria, depending on the content.
 
+- The definition of [disabled element][disabled] assumes that the presence of the `aria-disabled` attribute is matched by similar techniques for users not relying on assistive technologies. If this is not the case, that definition may produce incorrect results and in consequence this rule might be Inapplicable to some text nodes that still require a good contrast ratio.
```

Comment:
This phrase "matched by similar techniques for users not relying on assistive technologies" is a little vague. Can you clarify it?

Reply:
Bad copy/paste from the def 😞 Corrected.